packages/main/src/Calendar.js

```diff
@@ -143,52 +165,68 @@ const metadata = {
 /**
  * @class
  *
+ * <h3 class="comment-api-title">Overview</h3>
```

Comment:
The info about the detail is missing on the event description in the metadata. You can check other components for the syntax, but overall you need to update it to:
```js
"selected-dates-change": {
   detail: {
	dates: { type: Array },
   }
},
```

Reply:
Done